### PR TITLE
Added cast of case_tags to list

### PR DESCRIPTION
**Pull request by orangejenny**
https://dimagi-dev.atlassian.net/browse/REACH-112

Relevant: https://github.com/dimagi/commcare-hq/blob/d376be102c804b51cfe0695aef5db756e487b629/corehq/apps/app_manager/helpers/validators.py#L902-L905

Guessing this is a long-standing issue and there just aren't many apps that would trigger it: advanced modules, multiple open subcase actions, checking `case_tags` for the same tag twice.

Feature flag: advanced modules

### CODE REVIEW

**Comment by orangejenny** [comment]
or @emord

**Comment by emord** [comment]
Did you meant to link to `get_case_tags` defnition? looks like you just linked to the exact part of the code you changed: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/models.py#L643-L645

File: corehq/apps/app_manager/helpers/validators.py [hunk 1]
```diff
@@ -899,7 +899,7 @@ def check_actions(self):
         from corehq.apps.app_manager.models import AdvancedOpenCaseAction, LoadUpdateAction
 
         for action in self.form.actions.get_subcase_actions():
-            case_tags = self.form.actions.get_case_tags()
+            case_tags = list(self.form.actions.get_case_tags())
```
Comment (emord) [inline corehq/apps/app_manager/helpers/validators.py]: should this be outside of the for loop so that its not recalculated for every action?

**Comment by orangejenny** [comment]
@emord yes, I pasted the wrong link.

File: corehq/apps/app_manager/helpers/validators.py (hunk 1 above)
Comment (orangejenny) [reply corehq/apps/app_manager/helpers/validators.py]: good point, 88def0fbd20

**Review by emord — APPROVED** [review]

**Pull request merged**